java/src/main/java/com/powsybl/python/PyPowsyblApiLib.java

```diff
@@ -331,6 +343,25 @@ private static LoadFlowParameters createLoadFlowParameters(boolean dc, LoadFlowP
                 .setCountriesToBalance(CTypeUtil.toStringList(loadFlowParametersPtr.getCountriesToBalance(), loadFlowParametersPtr.getCountriesToBalanceCount())
                         .stream().map(Country::valueOf).collect(Collectors.toSet()))
                 .setConnectedComponentMode(LoadFlowParameters.ConnectedComponentMode.values()[loadFlowParametersPtr.getConnectedComponentMode()]);
+        Properties map = mergeListsToMap(keys, values, size);
+        bindingLoadFlowParametersExtensions(loadFlowParameters, map);
+        return loadFlowParameters;
+    }
+
+    private static void bindingLoadFlowParametersExtensions(LoadFlowParameters lfParams, Properties maps) {
+        // add extensions
+        // TODO use provider name to bind correspond extension
+        InMemoryPlatformConfig inMemoryPlatformConfig = new InMemoryPlatformConfig(FileSystems.getDefault());
+        MapModuleConfig hades2Config = inMemoryPlatformConfig.createModuleConfig("hades2-default-parameters");
+        MapModuleConfig olfConfig = inMemoryPlatformConfig.createModuleConfig("open-loadflow-default-parameters");
+        maps.forEach((k, v) -> {
+            hades2Config.setStringProperty((String) k, (String) v);
+            olfConfig.setStringProperty((String) k, (String) v);
+        });
+        ADNLoadFlowParameters h2ParamExt = new ADNLoadFlowParameters.ADNParametersConfigLoader().load(inMemoryPlatformConfig);
```

Comment:
We have to find a way to not to have an explicit dependency to ADN or OLF. Otherwise that won't work again when hades2 will be removed from OSS release.